app/src/main/java/com/example/activeamigo/PreferenceActivity.java

```diff
@@ -0,0 +1,333 @@
+package com.example.activeamigo;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.ActionBar;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.app.Activity;
+import android.app.DatePickerDialog;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.DatePicker;
+import android.widget.EditText;
+import android.widget.RadioButton;
+import android.widget.RadioGroup;
+import android.widget.Spinner;
+import android.widget.Toast;
+
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.firebase.firestore.DocumentReference;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+public class PreferenceActivity extends AppCompatActivity {
+
+    protected EditText dobEditText;
+    protected Button edit_save;
+    protected Spinner exercise_choice;
+    protected Spinner location_choice;
+    protected RadioGroup genderGroup;
+    protected String gender;
+    protected EditText bio;
+
+    protected String collection="Accounts";
```

Comment:
please make static

Reply:
needs to be private and modifiable. cannot be static